apps/ecommerce/lambda/src/routers/apiRouter.ts

```diff
@@ -1,8 +1,8 @@
 import { Router } from 'express';
-import { ApiController } from '../controllers';
+import { ApiController, ShopifyController } from '../controllers';
 const app = Router();
 
 app.get('/ping', ApiController.ping);
-app.get('/product/:id', ApiController.product);
+app.post('/resource/:resourceType/:id', ApiController.resource);
```

Comment:
my thoughts here: we should just drop these path params since all of the data we need is in the post body. it feels too redundant

Reply:
true, i agree. to my thinking since we really need to take a step back and consider the API design (endpoints, methods and schemas together) we could punt on this until later.